gnomad/utils/plotting.py

```diff
@@ -215,11 +215,14 @@ def plot_multi_hail_hist(
 
         hist_source = ColumnDataSource(data)
 
+        # fmt: off
```

Comment:
so the `# fmt: off` and `# fmt: on` are to disable the Black check for the lines in between them? What is the reason that`# pylint: disable=unsubscriptable-object` specifically has to be on this line rather than above (line 218) like done for the `# pylint: disable=invalid-unary-operand-type` in the other file?

Reply:
> so the # fmt: off and # fmt: on are to disable the Black check for the lines in between them? 
Yes, it doesn't format lines between `# fmt: off` and `# fmt: off`. https://black.readthedocs.io/en/stable/the_black_code_style.html

> What is the reason that # pylint: disable=unsubscriptable-object specifically has to be on this line rather than above (line 218) like done for the # pylint: disable=invalid-unary-operand-type in the other file?
It doesn't. I fixed this message first and it was only during fixing the the `invalid-unary-operand-type` messages that I found out about [block disables](http://pylint.pycqa.org/en/latest/user_guide/message-control.html#block-disables). Will change.